composer.json

```diff
@@ -50,7 +49,7 @@
 	"config": {
 		"sort-packages": true,
 		"platform": {
-			"php": "7.0.10"
+			"php": "7.2"
```

Comment:
This is the important bit to fix the composer resolution/download issue leading to the new failure.